CVE-2021-34866.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
Expected location (2021/34xxx/CVE-2021-34866.json) of file does not match actual location (CVE-2021-34866.json)

Reply:
moved to proper location and squashed commit